### PR TITLE
create configmap for configuration

### PR DESCRIPTION
**Pull request by jeanmi151**

### CODE REVIEW

File: opendata-ogc-features/values.yaml [hunk 1]
```diff
@@ -44,3 +44,4 @@ environment:
         key: password
         name: georchestra-database-geodata-secret
 tolerations: []
+ogcFeatConfig: []
```
Comment (pmauduit) [inline opendata-ogc-features/values.yaml:47]: should be an object, not an array ? `{}` instead ?

File: opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{{- if .Values.ogcFeatConfig }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  annotations:
+  labels: opendata-ogc-features
+  name: ogc-feat-config-cm
+data:
+  application.yaml: |
+    {{- .Values.ogcFeatConfig | nindent 4 }}
```
Comment (pmauduit) [inline opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml:10]: missing a ` | toYaml ` ?

File: opendata-ogc-features/values.yaml (hunk 1 above)
Comment (jeanmi151) [reply opendata-ogc-features/values.yaml:47]: good catch

File: opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml (hunk 2 above)
Comment (jeanmi151) [reply opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml:10]: seems to work like that need a change?

Comment (pmauduit) [reply opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml:10]: from what I remember I had to cast the value as yaml in an other context, but if it works, i guess it's fine

Comment (jeanmi151) [reply opendata-ogc-features/templates/opendata-ogc-features-config-cm.yaml:10]: actually the ` | toYaml ` add a  ` |- ` not wanted :) so the good way is as I did

**Comment by jeanmi151** [comment]
tested and working in a cluster

**Pull request merged**